TTHAnalysis/python/tools/nanoAOD/addTnpTree.py

```diff
@@ -6,6 +6,30 @@
 from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection 
 from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
 
+
+from CMGTools.TTHAnalysis.tools.nanoAOD.susySOS_modules import calculateRawMVA, SOSTightID2018, susyEleIdParametrization, VLooseFOEleID, tightEleID, clean_and_FO_selection_SOS
+conf = dict(muPt = 3.0, elePt = 5.0, sip3d = 2.5, dxy =  0.05, dz = 0.1, minMet = 50.0, ip3d = 0.0175, iperbolic_iso_0 = 20.0, iperbolic_iso_1 = 300.0)
+
+muonSelection = lambda l : abs(l.eta) < 2.4 and l.pt > conf["muPt"]  and l.sip3d < conf["sip3d"] and abs(l.dxy) < conf["dxy"] and abs(l.dz) < conf["dz"]  and l.pfRelIso03_all*l.pt < ( conf["iperbolic_iso_0"]+conf["iperbolic_iso_1"]/l.pt) and abs(l.ip3d) < conf["ip3d"]
+electronSelection = lambda l : abs(l.eta) < 2.5 and l.pt > conf["elePt"]  and l.sip3d < conf["sip3d"] and abs(l.dxy) < conf["dxy"] and abs(l.dz) < conf["dz"] and l.pfRelIso03_all*l.pt < ( conf["iperbolic_iso_0"]+conf["iperbolic_iso_1"]/l.pt) and abs(l.ip3d) < conf["ip3d"]
```

Comment:
It seems to me that also conf, muonSelection and electronSelection can be imported from susySOS_modules, avoiding duplication of these definitions.

Reply:
Ciao @vtavolar,
You are right, it is much better to import as much as we can.

To that end and following up on our private conversation, I modified the _susySOS_modules.py_ file (in [b953ff5](https://github.com/cms-susy-soft-leptons/cmgtools-lite/pull/6/commits/b953ff56aa9e35f80d1a9036529113739c272190)) to make all of the IDs importable by other modules. I now use the imported IDs in the TnP module (implemented in [b8d9442](https://github.com/cms-susy-soft-leptons/cmgtools-lite/pull/6/commits/b8d944257e055989fe48f649163646131d985526)).

Could you please review the changes and tell me if everything is fine?
On my end, I ran an example TnP tree production job and the result seems reasonable.

Reply:
Thanks a lot @VourMa for implementing these changes. I run a small local test to check that friend trees remain identical after this PR and that is the case. Merging.